backend/groq/summary_to_dialogue.py

```diff
@@ -7,32 +7,36 @@
 output: ./output/dialogue/*.txt
 """
 
+# Change imports
 import os
 import sys
 import argparse
 import time
 from dotenv import load_dotenv
-from llamaapi import LlamaAPI
+from groq import Groq
 import glob
 
-ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
+ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
 sys.path.append(ROOT)
 
-from utils.llama_api_helpers import make_api_call
-from utils.decorators import timeit
+from .utils.llama_api_helpers import make_api_call
+
+from .utils.decorators import timeit
 
 INPUT_DIR = os.path.join(ROOT, "output", "summaries")
 OUTPUT_DIR = os.path.join(ROOT, "output", "dialogue")
 os.makedirs(OUTPUT_DIR, exist_ok=True)
 
+# Change API setup
 load_dotenv()
-LLAMA_API_KEY = os.getenv('LLAMA_API_KEY')
-if not LLAMA_API_KEY:
-    raise ValueError("LLAMA_API_KEY environment variable not set")
-llama = LlamaAPI(LLAMA_API_KEY)
+GROQ_API_KEY = os.getenv('GROQ_API_KEY')
+if not GROQ_API_KEY:
```

Comment:
```python
GROQ_API_KEY = os.getenv('GROQ_API_KEY', None)
if GROQ_API_KEY is None:
```
I'd do the direct `None` check. This code would probably work most of the time though.